Jenkinsfile_CNP

```diff
@@ -37,6 +37,11 @@ def secrets = [
         secret('home-office-glasgow-email', 'IA_HOME_OFFICE_GLASGOW_EMAIL'),
         secret('home-office-apc-email', 'IA_HOME_OFFICE_APC_EMAIL'),
 
+        secret('ftpa-home-office-bradford-email', 'IA_FTPA_HOME_OFFICE_BRADFORD_EMAIL'),
+        secret('ftpa-home-office-taylor-house-email', 'IA_FTPA_HOME_OFFICE_TAYLOR_HOUSE_EMAIL'),
+        secret('ftpa-home-office-north-shields-email', 'IA_FTPA_HOME_OFFICE_NEWCASTLE_EMAIL'),
```

Comment:
secret variable is north-shields but environment variable is NEWCASTLE.
Would have been better to keep the same name for association

Reply:
This has been the case for all other environment variables, so I kept it the same to avoid confusion .